src/tests/json-strings/validationStrings.ts

```diff
@@ -722,3 +722,204 @@ export const documentChoiceDefaultBeforeChoice = `{
+          "Next": "NextState"
+      },
+      "DefaultState": {
+          "Type": "Fail",
+          "Error": "DefaultStateError",
+          "Cause": "No Matches!"
+      },
+      "NextState": {
+          "Type": "Pass",
+          "End": true
+      }
+  }
+}`
+
+export const documentInvalidPropertiesCatch = `{
+    "StartAt": "HelloWorld",
+    "States": {
+        "HelloWorld": {
+            "Type": "Task",
+            "Resource": "arn:aws:lambda:us-east-1:111111111111:function:FUNCTION_NAME",
+            "Catch": [
+                {
+                    "ErrorEquals": [
+                        "CustomError"
+                    ],
+                    "Next": "CustomErrorFallback",
+                    "OneInvalid": "something"
+                },
+                {
+                    "ErrorEquals": [
+                        "States.TaskFailed"
+                    ],
+                    "Next": "ReservedTypeFallback",
+                    "TwoInvalid": "something",
+                    "ThreeInvalid": "something"
+                },
+                {
+                    "ErrorEquals": [
+                        "States.ALL"
+                    ],
+                    "Next": "CatchAllFallback"
+                }
+            ],
+            "End": true
+        },
+        "CustomErrorFallback": {
+            "Type": "Pass",
+            "Result": "This is a fallback from a custom lambda function exception",
+            "End": true
+        },
+        "ReservedTypeFallback": {
+            "Type": "Pass",
+            "Result": "This is a fallback from a reserved error code",
+            "End": true
+        },
+        "CatchAllFallback": {
+            "Type": "Pass",
+            "Result": "This is a fallback from a reserved error code",
+            "End": true
+        }
+    }
+}`
+
+export const documentInvalidPropertiesChoices = `{
+  "Comment": "An example of the Amazon States Language using a choice state.",
+  "StartAt": "FirstState",
+  "States": {
+      "FirstState": {
+          "Type": "Task",
+          "Resource": "arn:aws:lambda:us-east-1:111111111111:function:FUNCTION_NAME",
+          "Next": "ChoiceState"
+      },
+      "ChoiceState": {
+          "Type": "Choice",
+          "Choices": [
+              {
+                  "Not": {
+                      "Variable": "$.foo",
+                      "StringEquals": "blabla",
+                      "NumericGreaterThanEquals": 20,
+                      "FirstInvalidProp": {}
+                  },
+                  "Next": "FirstMatchState"
+              },
+              {
+
+                  "SecondInvalidProp": {},
+                  "And": [
+                      {
+                          "Not": {
+                              "Variable": "$.foo",
+                              "StringEquals": "blabla",
+                              "ThirdInvalidProp": {}
```

Comment:
Add additional coverage for no "Next" in nested choice rules

Reply:
done